tox.ini

```diff
@@ -140,5 +140,15 @@ basepython =
     python2.7
 commands =
     python {toxinidir}/system_tests/run_emulator.py --package=pubsub
-passenv = GOOGLE_CLOUD_*
+passenv =
+    GOOGLE_CLOUD_DISABLE_GRPC
+deps = {[testenv:datastore-emulator]deps}
+
+[testenv:bigtable-emulator]
+basepython =
+    python2.7
+commands =
+    python {toxinidir}/system_tests/run_emulator.py --package=bigtable
+passenv =
+    GOOGLE_CLOUD_DISABLE_GRPC
 deps = {[testenv:datastore-emulator]deps}
```

Comment:
Maybe we should have a `[testenv:service-emulators]` that these common deps could pull from? Could also use this to run all the emulator tests...I'm not sure how useful running all emulator tests at once would be though.